cluster/config-defaults.yaml

```diff
@@ -23,6 +23,10 @@ skipper_ingress_max_replicas: "10"
 skipper_ingress_min_replicas: "1"
 skipper_ingress_cpu: "500m"
 skipper_ingress_memory: "500Mi"
+{{if eq .Environment "production"}}
```

Comment:
Please add a default for both environments (set to `false` in production) and then check for `if eq ... "true"`. It's very confusing otherwise, because setting it to `false` wouldn't actually disable anything (and in fact it's impossible to disable in test clusters).

Reply:
ok